package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha  --require @babel/register ./tests/* --exit"
```

Comment:
You have not installed @babel/register

Reply:
@babel/register is being installed by another chore at the moment